nix/impure-shell.nix

```diff
@@ -90,7 +90,7 @@ mkShell {
 
   postVenvCreation = ''
     unset SOURCE_DATE_EPOCH
-    ( cd server ; python -m pip install --no-dependencies -e . )
+    ( cd server ; python -m pip install --no-build-isolation --no-dependencies -e . )
```

Comment:
This is to avoid downloading torch as a build dependency (since without build isolation it is used from the environment).